src/main/java/com/odysseusinc/arachne/datanode/service/impl/DataSourceServiceImpl.java

```diff
@@ -126,8 +126,8 @@ public DataSource create(User owner, DataSource dataSource) throws NotExistExcep
     @Override
     public void createOnCentral(User owner, DataSource dataSource) {
 
+        AutoDetectedFields autoDetectedFields = autoDetectFields(dataSource);
         if (Objects.equals(NETWORK, dataNodeService.getDataNodeMode())) {
```

Comment:
just `==`

Reply:
Just to not fallen into the Holy Wars. And you didn't find another one easter egg)